crates/cargo-wdk/tests/emulated-workspace/umdf-driver-workspace/Cargo.lock

```diff
@@ -47,17 +47,18 @@ version = "1.1.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "6d36fc52c7f6c869915e99412912f22093507da8d9e942ceaf66fe4b7c14422a"
 dependencies = [
- "windows-sys",
+ "windows-sys 0.52.0",
```

Comment:
`windows-sys` 0.52.0 seems to have creeped back in.

Reply:
okay removed it and also removed the extra windows-targets dupes